.github/workflows/schema-check.yml

```diff
@@ -23,13 +23,14 @@ jobs:
         with:
           files: |
             **/*.ifc
-          files_separator: ";"
+          separator: "§"
```

Comment:
I think you can reuse this "separator" attribute. Cannot remember how. Probably by full path. 🤔

Reply:
Resolved in call.